content/docs/start/data-pipelines.md

```diff
@@ -260,22 +261,23 @@ Stage 'prepare' didn't change, skipping
 Stage 'featurize' didn't change, skipping
 ```
 
-Same as before, no need to run `prepare`, `featurize`, etc ... but, it doesn't
-run even `train` again this time either! It cached the previous run with the
-same set of inputs (parameters + data) and reused it.
+This is the same result as before (no need to rerun `prepare`, `featurize`,
+etc.) but it also doesn't run `train` again this time either! DVC maintains a
```

Comment:
Good rephrase, thanks. But "also doesn't run `train` again this time either" is still kind of unreadable.

Reply:
"again/this time/either" technically repeats sentiment thrice but it seems to be the important message the paragraph was trying to drive home and may be easy to miss sans emphasis...

How about "... before. No need to rerun `prepare`, `featurize`, etc. but also -- once again -- no need to rerun `train`"

Reply:
Yeah I guess "this is **not** the same result as before" then. Simplification:
```suggestion
Like before, there's no need to rerun `prepare`, `featurize`,
etc. But it doesn't run `train` this time either! DVC maintains a
```

Reply:
changed